### ISSUE TITLE
Results on testing data look odd

### ISSUE THREAD

**Issue by ShreyasSkandan** [JiaRenChang/PSMNet#85]
Hi,

I'm trying to run your pre-trained KITTI 2015 model on the 2015 test set but the results don't look right.

Do you have any idea what might be going wrong?

There are some examples:

![000165_10](https://user-images.githubusercontent.com/5595336/45967299-93131d00-bffb-11e8-8d14-5bced16f16db.png)
![000130_10](https://user-images.githubusercontent.com/5595336/45967300-93131d00-bffb-11e8-8021-7b5e43cb9191.png)
![000110_10](https://user-images.githubusercontent.com/5595336/45967301-93131d00-bffb-11e8-9bff-70eb00f69cc3.png)
![000095_10](https://user-images.githubusercontent.com/5595336/45967302-93131d00-bffb-11e8-8674-91ab6db81471.png)
![000078_10](https://user-images.githubusercontent.com/5595336/45967303-93131d00-bffb-11e8-95e0-b0bbe2e47bcd.png)

I haven't made any modifications to your original code, except for adding a path to the testing data folder and a path to the model (that I downloaded from the google drive link on your repo landing page).

Thanks in advance,

SS

**Comment by yifaliu** [comment]
I have the same issue. I just run the submission.py using the KITTI 2015 testing dataset and pretrained_model_KITTI2015.tar. Results look weird. I didn't modify anything.

**Comment by yifaliu** [comment]
When I searched more within these issues, I got the answer.
The torchvision version should be 0.2.0 (not 0.2.1)
Just pip install torchvision=0.2.0 and can get the reasonable result after running the submission.py

**Comment by ShreyasSkandan** [comment]
> When I searched more within these issues, I got the answer.
> The torchvision version should be 0.2.0 (not 0.2.1)
> Just pip install torchvision=0.2.0 and can get the reasonable result after running the submission.py

That did it! Thank you.
